docs/features/DefaultInterfaceImplementation.md

```diff
@@ -27,6 +27,41 @@ class Test1 : I1
 }
 ```
 
+- Re-abstraction of interface implementation in derived interfaces. Types implementing the derived interface are required to supply implementation even when a base interface provides an implementation. 
+Here is an example:
+```
+public interface I1
+{
+    void M1() 
+    {
+    }
+
+    int P1 
+    {
+        get => throw null;
+        set => throw null;
+    }
+
+    event System.Action E1
+    {
+        add => throw null;
+        remove => throw null;
+    }
+}
+
+public interface I2 : I1
+{
+    abstract void I1.M1();
+    abstract int I1.P1 {get;set;}
+    abstract event System.Action I1.E1;
+}
+
+class Test1 : I2 // This type must implement all members of I1
+{
+}
+```
```

Comment:
nit: If I recall, this is getting a non-obvious representation in metadata. Consider documenting too (helps in case of cross-language support).